From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Wed, 17 Nov 2021 07:22:04 +0100
Subject: [ARCHEOLOGY] fix sound HDMI rk3399 and bootsplash (#3260)

> X-Git-Archeology: - Revision 21d5340b35f43dda30c86320433ab3fb82997594: https://github.com/armbian/build/commit/21d5340b35f43dda30c86320433ab3fb82997594
> X-Git-Archeology:   Date: Wed, 17 Nov 2021 07:22:04 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: fix sound HDMI rk3399 and bootsplash (#3260)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2813365dd25e3ad110936cbf014b95b38d7090ec: https://github.com/armbian/build/commit/2813365dd25e3ad110936cbf014b95b38d7090ec
> X-Git-Archeology:   Date: Mon, 07 Nov 2022 21:29:00 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move known non working rockhip64 boards to previous boot loader (#4392)
> X-Git-Archeology:
---
 include/configs/evb_rk3399.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/include/configs/evb_rk3399.h b/include/configs/evb_rk3399.h
index 492b7b4df1..6320914207 100644
--- a/include/configs/evb_rk3399.h
+++ b/include/configs/evb_rk3399.h
@@ -6,11 +6,6 @@
 #ifndef __EVB_RK3399_H
 #define __EVB_RK3399_H
 
-#define ROCKCHIP_DEVICE_SETTINGS \
-		"stdin=serial,usbkbd\0" \
-		"stdout=serial,vidconsole\0" \
-		"stderr=serial,vidconsole\0"
-
 #include <configs/rk3399_common.h>
 
 #define SDRAM_BANK_SIZE			(2UL << 30)
-- 
Armbian

